kwanghwi/myapp/db/migrations/20221220062637_create_users_table.sql

```diff
@@ -0,0 +1,13 @@
+-- migrate:up
+CREATE TABLE users (
+  id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  userId VARCHAR(200) NOT NULL,
+  password VARCHAR(200) NOT NULL,
+  email VARCHAR(200) NOT NULL,
+  profile_image VARCHAR(1000) NULL,
+  created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP
+);
```

Comment:
1. 컬럼명은 camelCase 혹은 snake_case 중 하나로 선택하여 지정할 수 있습니다.
다만 그 방식이 통일되어야 데이터베이스를 사용하는 다른 개발자들에게 혼란을 최소화할 수 있을 것입니다.
camelCase 혹은 snake_case로 컬럼명을 모두 통일해주세요!

2. 동일한 유저 데이터가 중복 저장이 되지 않도록 email 컬럼에 Unique 제약 조건을 추가해주세요!
3. updated_at 컬럼은 데이터가 수정된 시점의 시간을 반영할 수 있도록 아래와 같이 수정해주세요!

```suggestion
CREATE TABLE users (
  id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
  user_id VARCHAR(200) NOT NULL,
  password VARCHAR(200) NOT NULL,
  email VARCHAR(200) NOT NULL,
  profile_image VARCHAR(1000) NULL,
  created_at TIMESTAMP NOT NULL DEFAULT NOW(),
  updated_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP,
  CONSTRAINT users_ukey UNIQUE (email)
);
```